impl/src/main/java/org/jboss/cdi/tck/tests/inheritance/specialization/qualifiers/StaticNestedClassesParent.java

```diff
@@ -19,14 +19,15 @@
 import jakarta.enterprise.context.Dependent;
 import jakarta.enterprise.inject.Specializes;
 
-@Dependent
 public class StaticNestedClassesParent {
     @Mock
+    @Dependent
     public static class StaticSpecializationBean {
 
     }
 
     @Specializes
+    @Dependent
```

Comment:
`@Specializes` is not a bean defining annotation, so without it, you won't even discover the bean.

Reply:
Ah true. I thought it would be defaulted, but there's needs to be a BDA for sure.

In `StaticSpecializationBean` above, `@Mock` would serve as a BDA though, right?

...

OK, wrong. I thought it's a stereotype, but it's a qualifier. So both have to have a BDA.